reviews/index.html

```diff
@@ -114,8 +114,8 @@ <h2 class="notoc">Filter results</h2>
 			var totals=0
 			var counter=maxpages
 			
-			function getAllData () {
-				for (var p=1;p<maxpages+1;p++) fetchIssues(p)
+			function getAllData (maxpages_new) {
+				for (var p=1;p<maxpages_new+1;p++) fetchIssues(p)
```

Comment:
Use a better name? Perhaps `pages`? Or `totalPages`? Why don't we just replace the value of `maxpages` with a call to `fetchIssuesCount()`?

Reply:
hrm.. since this PR was something of quick hack, so to remove old variable, I think I should rewrite this part with `Promise.all`.
Please wait a bit more for rewriting.

Reply:
Ping